js-mentoring-program-master/tasks/functions/functions.js

```diff
@@ -89,10 +117,10 @@ const myObject = {
 	age: 25,
 	friends: ['Mike', 'Alan', 'Daniel'],
 	keys() {
-		//write your code here
+		console.log(Object.keys(this));
 	},
 	call() {
-		//write your code here
+		return 'My name is' + ' ' + this.name + ' ' + this.lastName + ' ' + 'I am ' + ' ' + this.age + ' ' + 'years old. My best friend is' + ' ' + this.friends[2];
```

Comment:
Refactor this code using https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/template_strings